### PR TITLE
[TR] Getting Songs From Video Games

### PR DESCRIPTION
**Pull request by Lxran**
Added the Turkish translation of the "Getting Songs From Video Games" article.

### CODE REVIEW

File: wiki/Guides/Getting_Songs_From_Video_Games/tr.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Öncelikle bilgisayarınıza gerekli uyguluamaları indirin ve kurun (eğer hala kurmadıysanız). Onları kurduktan sonra, foobar2000'i ve Windows Gezginini çıkarttığınız yedeğin müzik klasöründe açın.
+
+*Not: Oyundan oyuna klasörün adı değişebilir. Müzik klasörü için yaygın isimler: **stream, sound, bgm, music**.*
+
+*Not 2: Bazı oyunlar motor kısıtlamaları/geliştirici yakınmaları/herhangi bir sebepten ötürü farklı bir dosya uzantısı kullanabilir ya da hiç dosya uzantısı kullanmayabilir. Genellikle bir Google araması bu tür oyunlar hakkında sizlere bilgi verebilir.*
+
+## Şarkımızı yüklemek
+
+Oyunun müzik klasörüne girdiğinizde, şarkıları foobar2000'e yükleme zamanı geldi demektir. Bunu yapmak için, klasördeki tüm dosyaları seçin ve foobar'ın ekranına sürükleyin. Bütün şarkılar yüklendikten sonra, her birini aradığınız şarkıyı bulana kadar dinleyin. Bu kılavuzda, *Chapter 3 Boss Battle Theme* şarkısını arıyoruz (Hala oyunu oynayanlar için sonunu mahvetmemek için söyleyemiyoruz :p) o yüzden aradığımız dosyanın adı **btl\_gns1\_32k.stm**. Şarkıyı dinlemekten çekinmeyin.
```
Comment (zeusminus) [inline wiki/Guides/Getting_Songs_From_Video_Games/tr.md:26]:
```suggestion
Oyunun müzik klasörüne girdiğinizde, şarkıları foobar2000'e yükleme zamanı geldi demektir. Bunu yapmak için, klasördeki tüm dosyaları seçin ve foobar'ın ekranına sürükleyin. Bütün şarkılar yüklendikten sonra, her birini aradığınız şarkıyı bulana kadar dinleyin. Bu kılavuzda, *Chapter 3 Boss Battle Theme* şarkısını arıyoruz (Hala oyunu oynayanların hatrına sürprizi bozmamak için gerçek ismini söyleyemeyiz :p) o yüzden aradığımız dosyanın adı **btl\_gns1\_32k.stm**. Şarkıyı dinlemekten çekinmeyin.
```

File: wiki/Guides/Getting_Songs_From_Video_Games/tr.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Dönüştürme
+
+Dönüştüreceğiniz şarkıya sahip olduktan sonra, çalma listesinde şarkıya sağ tıklayın, *Convert*'i seçin ve ardından *...* seçeğini seçin. Bu size dönüşümün nasıl yapılacağınızı seçmenizi sağlayan bir diyalog ekranını getirir.
```
Comment (zeusminus) [inline wiki/Guides/Getting_Songs_From_Video_Games/tr.md:30]:
```suggestion
Dönüştüreceğiniz şarkıya sahip olduktan sonra, çalma listesinde şarkıya sağ tıklayın, *Convert*'i seçin ve ardından *...* seçeneğini seçin. Bu size dönüşümün nasıl yapılacağınızı seçmenizi sağlayan bir diyalog ekranını getirir.
```
fix typo

File: wiki/Guides/Getting_Songs_From_Video_Games/tr.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Bu liste osu! için doğru dönüştürme ayarlarını sıralar :
```
Comment (zeusminus) [inline wiki/Guides/Getting_Songs_From_Video_Games/tr.md:32]:
```suggestion
Bu liste osu! için doğru dönüştürme ayarlarını sıralar:
```

File: wiki/Guides/Getting_Songs_From_Video_Games/tr.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+- **Output Format:** Format: MP3 (LAME), V2 *(Bitrate kendini 190 kbps olarak ayarlayacak)*
+- **Destination:** Output folder: source file folder. Output type: tracks into individual files. File name pattern: %title%
+- **Processing:** Hard -6dB limiter. *(İsteğe bağlı)*
+- **Other:** When finished: do nothing.
+
+*Not: Eğer daha önce foobar2000'de MP3 dönüştürmediyseniz, dönüştürücü sizden **Lame.exe** dosyasını bulmanızı isteyecektir. **Lame.exe** dosyasını çıkarttığınız klasöre gidin ve dosyayı seçin.*
```
Comment (zeusminus) [inline wiki/Guides/Getting_Songs_From_Video_Games/tr.md:39]:
```suggestion
*Not: Eğer daha önce foobar2000'de MP3 dönüştürmediyseniz, dönüştürücü sizden **lame.exe** dosyasını konumlandırmanızı isteyecektir. **lame.exe** dosyasını çıkarttığınız klasöre gidin ve dosyayı seçin.*
```

File: wiki/Guides/Getting_Songs_From_Video_Games/tr.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+Ayarlarınızı yapmayı bitirdiğinizde, *Save* butonuna tıklayın ve mevcut ön ayarı değiştirmek isteyip istemediğinizi sorduğunda *Save as new* seçeneğini tıklayın. Gelecekte başka şarkıları hızla dönüştürebilmek için ona bir isim verin. Dönüştürmeye hazır olduğunuzda dönüşümün başlaması için *Convert* butonuna tıklayın. Bittiğinde, çıkan dosyayı bulmak için klasörü tarayın. Bizim durumuzuda, bu dosya **btl\_gns1\_32k.mp3**.
```
Comment (zeusminus) [inline wiki/Guides/Getting_Songs_From_Video_Games/tr.md:41]:
```suggestion
Ayarlarınızı yapmayı bitirdiğinizde, *Save* butonuna tıklayın ve mevcut ön ayar üzerine yazıp yazmamak istediğinizi sorduğunda ise *Save as new* seçeneğine tıklayın. Gelecekte başka şarkıları hızla dönüştürebilmek için ona bir isim verin. Dönüştürmeye hazır olduğunuzda dönüşümün başlaması için *Convert* butonuna tıklayın. Bittiğinde, çıkan dosyayı bulmak için klasörü tarayın. Bizim durumuzuda, dosya ismi **btl\_gns1\_32k.mp3** idi.
```

File: wiki/Guides/Getting_Songs_From_Video_Games/tr.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+## Kapanış
```
Comment (zeusminus) [inline wiki/Guides/Getting_Songs_From_Video_Games/tr.md:43]:
```suggestion
## Bitirirken
```

File: wiki/Guides/Getting_Songs_From_Video_Games/tr.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
+# Video oyunlarından şarkılar nasıl alınır
+
+Bu kılavuz, başta PS2, GCN/Wii, Xbox 360, vs. gibi konsollardan olmak üzere, video oyunlarından bir şarkıyı isteyen mapperlar içindir. Taşınabilir ya da daha eski konsolları da içinde bulundurabilir (SNES, PS1, PSP, N64, NDS vs. bakınız) ancak onlardan müziği almak daha fazla şey gerektirir.
+
+## Başlamadan önce
+
+Bu kılavuz birkaç şeyi gerektirecek. Bunlar, onlara sahip değilseniz indirmeniz ya da onlar hakkında daha fazla bilgi sahibi olmanız için aşağıda sıralanmıştır:
```
[the remark this answers is not in the document]
Comment (Lxran) [reply wiki/Guides/Getting_Songs_From_Video_Games/tr.md:7]:
```suggestion
Bu kılavuz birkaç şeyi gerektirecek. Bunlar, onlara sahip olmamanız ihtimaline karşı onları indirebilmeniz ya da onlar hakkında daha fazla bilgi sahibi olabilmeniz için aşağıda sıralanmıştır:
```

I feel like this flows better and I don't believe there is a reason to complicate the sentence even further, as they both end up at the same meaning.

[the remark this answers is not in the document]
Comment (zeusminus) [reply wiki/Guides/Getting_Songs_From_Video_Games/tr.md:7]: fair, I don't see any issue applying your suggestion here

**Pull request merged**